readme.md

```diff
@@ -197,8 +197,8 @@ die Plenen selbst.
   einberufen werden.
   - Plenumspunkte sollen immer mind. 72h vor Beginn des Plenums auf der 
   Plenumswikiseite eingetragen sein.
-  - Abstimmvorschläge muessen immer mind. 72h vor Beginn des Plenums auf der 
-  Plenumswikiseite eingetragen sein.
+  - Abstimmvorschläge (nicht im Wortlaut, sondern im Topic) muessen immer
```

Comment:
Was soll ein Abstimmungsvorschlag im Topic sein? In welchem Topic? Und was bringt es wenn er nicht im Wortlaut ist? Ist ein Plenumspunkt ueber den diskutiert wird nicht ausreichend? Oder wieso kann ueber normale Punkte keine Abstimmung stattfinden wenn es keinen Konsens gibt?

Reply:
Die Idee von solchen Regeln ist doch, dass man aus der Einladung sehen kann, welche Themen diskutiert (und dann eben abgestimmt) werden sollen, so dass niemand überrascht ist, wenn auf einem Plenum Dinge entschieden wurden, die einem auch wichtig gewesen wären, man aber nicht da war, da man nicht ahnen konnte, dass sie behandelt werden. 

Der Nachsatz soll meiner Meinung nach nur sicher stellen, dass die genaue Abstimmungsfrage, über die abgestimmt wird, nicht auch schon in der Einladung stehen muss, sondern sich noch aus der Diskussion ergeben kann. Beispiel: Tagesordnung sagt "Beamer Lampe ist durchgebrannt, was wollen wir tun?", dann kann auf dem Plenum trotzdem über die Frage abgestimmt werden "Sollen wir die Ersatzlampe für 42 Euro kaufen oder lieber gleich einen moderneren Beamer für 1337 Euro", aber nicht "lass uns einen zweiten Matematen für den Keller anschaffen und dafür die Mitgliedsbeiträge verdoppeln". Klar ist das eine Grauzone, aber wir sind doch erwachsen.

Reply:
Jo. "im Topic" verstehe ich als, das Thema über das abgestimmt wird muss vorab klar sein. Habe bei dem PR nur den Wortlaut aus dem Protokoll übernommen. Falls wer eine bessere Formulierung hat, gerne her damit

Reply:
```suggestion
  - Abstimmvorschläge (nicht im Wortlaut, sondern im Themenfeld) müssen immer
```